tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -68,37 +68,44 @@ Verify GPU Operator Deployment  # robocop: disable
     # ...   nvidia-operator-validator DS
 
 Verify That Prometheus Image Is A CPaaS Built Image
-    [Tags]    Sanity   
+    [Tags]    Sanity
     ...     Tier1
```

Comment:
When did we start using `Tier1` as a tag for TCs?

Reply:
@lugi0 we have to set TierX to all the tests (same as we do in Polarion)

Reply:
Since when?